libs/utils/test.py

```diff
@@ -98,12 +98,18 @@ def _experimentsInit(cls):
         """
         Code executed before running the experiments
         """
+        if cls.target.os == "android":
+            # Stop all the system services that introduce noise
```

Comment:
Maybe it's worth to log a warning about that?
Also, maybe we can add a target configuration feature to enable/disable this behaviour?
I'm thinking about the possibility to have, in the future, android based tests which requires the runtime to be up and running...